UPGRADE.md

```diff
@@ -2,6 +2,17 @@
 
 ## 2.5.0
 
+### JS Dependencies update
+
+Sulu requires now new versions of the following JS packages:
```

Comment:
would add this to the existing list once https://github.com/sulu/sulu/pull/6688 is merged